cmd/satellite/main.go

```diff
@@ -137,8 +142,8 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 
 	runError := peer.Run(ctx)
 	closeError := peer.Close()
-
-	return errs.Combine(runError, closeError, db.Close())
+	err = errs.Combine(err, runError, closeError)
+	return err
```

Comment:
`return errs.Combine(runError, closeError)`

Reply:
This will capture any error from the defer as well?

Reply:
defers run after the return, so the defer will capture this error.

Reply:
thanks!